### ISSUE TITLE
Add name of the library in report

### ISSUE THREAD

**Issue by WonderCsabo** [hierynomus/license-gradle-plugin#94]
I am not sure Gradle offers this information or not, but it would be nice to have the name of the library, which is added [like this](https://github.com/excilys/androidannotations/blob/develop/AndroidAnnotations/androidannotations-core/androidannotations-api/pom.xml#L30) in POM descriptors.

**Comment by hierynomus** [comment]
I dont think this is available in the dependency information. I'd have to look it up.

**Comment by WonderCsabo** [comment]
Since you are already [parse](https://github.com/hierynomus/license-gradle-plugin/blob/master/src/main/groovy/nl/javadude/gradle/plugins/license/LicenseResolver.groovy#L211) the `pom.xml` file, you can just obtain the `name` tag.
